app/MMB/src/scenes/tripServices/insurance/insuranceSelectionScene/PassengerInfo.js

```diff
@@ -14,7 +18,9 @@ const PassengerInfo = ({ title, fullName, birthday }: Props) => (
     </Text>
     <Text style={[styles.passengerText, styles.passengerBirthday]}>
       <Translation
-        passThrough={DateFormatter(new Date(birthday)).formatToBirthday()}
+        passThrough={
+          birthday && DateFormatter(new Date(birthday)).formatToBirthday()
```

Comment:
if birthday is null, you will actually output null as a string in this case
should be `{birthday !== null && <Text>..</Text>}`